processors/processor/src/main/kotlin/com/driver733/mapstructfluent/MapstructFluentExtensionsAnnotationProcessor.kt

```diff
@@ -31,6 +31,9 @@ class MapstructFluentExtensionsAnnotationProcessor : AbstractProcessor(), Mapper
         fileSpecBuilder(processingEnv, mapper, "FluentExtensions")
 
     override fun process(fileSpecBuilder: FileSpec.Builder, method: ExecutableElement, mapper: Element, src: String?) {
+        val receiver = method.parameters.first().kotlinType()
+        val otherParameters = method.parameters.drop(1).map(VariableElement::toParameterSpec)
```

Comment:
Common logic should be moved to the GenericProcessor class

Reply:
I've moved a little bit more code to GenericProcessor, please verify are you okay with that